pkg/api/v1/validation/validation.go

```diff
@@ -70,12 +70,16 @@ func ValidateResourceRequirements(requirements *v1.ResourceRequirements, fldPath
 
 func validateContainerResourceName(value string, fldPath *field.Path) field.ErrorList {
 	allErrs := validateResourceName(value, fldPath)
-	if len(strings.Split(value, "/")) == 1 {
-		if !helper.IsStandardContainerResourceName(value) {
-			return append(allErrs, field.Invalid(fldPath, value, "must be a standard resource for containers"))
-		}
+
+	// Container resources must be one of:
+	// - A known resource in the `*kubernetes.io/` namespace.
+	// - An extended resource (not in the `*kubernetes.io/` namespace.
+	// - An opaque integer resource [DEPRECATED] (TODO: Remove following deprecation)
+	if !helper.IsStandardContainerResourceName(value) &&
```

Comment:
Hey @smarterclayton -- thanks for taking a look at this. Did you see the related issue (#50658)? A related PR to allow any resource name outside of `*kubernetes.io/` to be used, so this would only break someone if they have resources prefixed with `...kubernetes.io/`. Non-namespaced resources were already properly rejected before this patch.